packages/env/lib/config/test/config.js

```diff
@@ -653,24 +682,24 @@ describe( 'readConfig', () => {
 			expect( config.env.development.mappings ).toEqual( {
 				test1: {
 					basename: 'test1',
-					path: '/test1',
+					path: resolve( '/test1' ),
```

Comment:
`resolve` may seem meaningless, but it is necessary to remove the drive letter (e.g. `C:`) on Windows OS.

Reply:
Could you add this as a comment in the code?